tests/UnitTests/CoreGraphics.drawing/CGContextDrawing_ClearRectTests.cpp

```diff
@@ -0,0 +1,101 @@
+                                              "1f,%0.1f,%0.1f,%0.1f,%0.1f].png"),
+                                        rect.origin.x,
+                                        rect.origin.y,
+                                        rect.size.width,
+                                        rect.size.height,
+                                        transformation.a,
+                                        transformation.b,
+                                        transformation.c,
+                                        transformation.d,
+                                        transformation.tx,
+                                        transformation.ty);
+    }
+};
+
+DRAW_TEST_P(CGClearRect, Transformed) {
+    CGRect rect = ::testing::get<0>(GetParam());
+    CGAffineTransform transformation = ::testing::get<1>(GetParam());
+
+    CGContextRef context = GetDrawingContext();
+
+    CGContextAddRect(context, CGRectMake(100, 50, 50, 50));
+    CGContextConcatCTM(context, transformation);
+    CGContextClip(context);
+
+    CGContextSetRGBFillColor(context, 0.48, 0.73, 0, 1);
+    CGContextClearRect(context, rect);
+}
+
+DRAW_TEST_P(CGClearRect, Transformed2) {
+    CGRect rect = ::testing::get<0>(GetParam());
+    CGAffineTransform transformation = ::testing::get<1>(GetParam());
+
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    CGContextAddRect(context, CGRectMake(0, 0, 250, 250));
+    CGContextConcatCTM(context, transformation);
+    CGContextClip(context);
+
+    CGContextSetRGBFillColor(context, 0.48, 0.73, 0, 1);
+    CGContextClearRect(context, rect);
+}
+
+static CGRect rects[] = { CGRectMake(0, 0, 100, 100), CGRectMake(0, 0, 50, 250), CGRectMake(100, 100, 125, 250) };
+static CGAffineTransform transformation[] = { CGAffineTransformMakeRotation(0.4),
+                                              CGAffineTransformMakeTranslation(3, 6),
+                                              CGAffineTransformMakeScale(0.25, 0.75),
+                                              CGAffineTransformMake(1.f, 0.f, 0.3f, 1.f, 0.f, 0.f),
+                                              CGAffineTransformIdentity };
+
+INSTANTIATE_TEST_CASE_P(CGContextTests, CGClearRect, ::testing::Combine(::testing::ValuesIn(rects), ::testing::ValuesIn(transformation)));
+
+DRAW_TEST_F(CGContext, ClearRect, WhiteBackgroundTest<>) {
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    CGContextSetRGBFillColor(context, 1.0, 0.0, 0.0, 1);
+    CGContextFillRect(context, bounds);
+
+    CGRect borderRect = CGRectInset(bounds, 30, 50);
+    CGContextClearRect(context, borderRect);
+}
+
+DISABLED_DRAW_TEST_F(CGContext, ClearRectArc, WhiteBackgroundTest<>) {
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    CGRect cirleRect = CGRectMake(0, 0, 100, 100);
+    CGContextAddArc(context, 50, 50, 50, 0.0, 2 * M_PI, 0);
```

Comment:
when we attempt to fill the geometry with an arc.
The fill is a success but the rendering doesn't show any signs of being filled.
I tried with a custom circle geometry of the same size and it works.
investigating, it looks like some issues with the arc geometry.